src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/ace/AceEditorBackgroundLinkHighlighter.java

```diff
@@ -351,6 +352,13 @@ private void navigateToUrl(String url)
       if (reSrcRef.test(url))
          return;
 
+      // github issues
+      Pattern reIssue = Pattern.create("^#[0-9]+$");
+      if (reIssue.test(url))
+      {
+         // TODO: globalDisplay_.openWindow(BugReports + /url)
```

Comment:
Not sure how to do this. Perhaps it should process `git remote -v` instead of reading the DESCRIPTION so that it does not have to be an R package ?

Reply:
The parentheses are part of the regex so this really looks at ˋ( + # + numbers + )ˋ inside a string, which does not clash with the rule for colors. 

I suppose this could be handled directly in the highlight rules as we do colors instead of what this pr does. 

I'll have another go.